data/src/main/java/com/teamagam/gimelgimel/data/location/LocationFetcher.java

```diff
@@ -133,6 +137,16 @@ public void requestLocationUpdates() {
         mLocationManager.addGpsStatusListener(mStoppedGpsStatusDelegator);
     }
 
+    @SuppressWarnings("MissingPermission")
+    private void requestLocationUpdates(String provider) {
+        try {
```

Comment:
what happens if it cannot add any provider? does the user knows those sort of things?
what if the users enables those options after the application initiated?